readme.md

```diff
@@ -19,7 +19,7 @@ If you have a link or a resource to add, you have a few options:
 Likewise, there are other ways to see how people are implementing IIIF.
 - [Github Topics page](https://github.com/topics/iiif?o=desc&s=updated) - This page will show you the Github repositories that self-identify as IIIF-related, with the most recently updated projects listed first. 
 - [IIIF Community on Zenodo](https://zenodo.org/communities/iiif/) - A Zenodo community for gathering articles and data related to IIIF.
-- [People, projects and institutions active within the IIIF Community on Mastodon](https://julsraemy.ch/mastodiiif/)
+- [IIIF on Mastodon](https://julsraemy.ch/mastodiiif/) - People, projects and institutions that self-identify as being active within the IIIF Community on  Mastodon, the social network that is part of the Fediverse.
```

Comment:
Ah -- I left a comment on a different commit, I think. (I do not comment lines often. My apologies.)
----
I like the first part, but the added ending raises more questions than it answers, at least to me. I'd prefer just ending at "Mastodon". Or maybe, if you want to explain Mastodon briefly, "Mastodon, a federated social network."